src/SQLAlchemyLibrary/query.py

```diff
@@ -204,9 +205,9 @@ def execute_sql_script(self, sqlScriptFileName, **named_args):
         DELETE
           FROM employee_table
         """
-        with open(sqlScriptFileName) as sqlScriptFile:
-            queries = self._split_sql_script(sqlScriptFile.read())
-            self._run_query_list(queries, **named_args)
+        batches = OperatingSystem().get_file(sqlScriptFileName)
```

Comment:
RF offers abstractions over implementations of operating systems, which I find reliable... They can (but in this case I should check implementation of this method in RF.OperatingSystem Library) also generate more RF-like logs on fail. I find consistent with this framework. Do You prefer Python-specific methods?